.github/workflows/ci.yml

```diff
@@ -113,11 +116,19 @@ jobs:
       run: make crypto/relic/build
     - name: Run tests
       if: github.actor != 'bors[bot]'
-      run: make -C crypto cross-blst-test
+      run: |
```

Comment:
Can't `lscpu` be run here before the `if` statement?

Reply:
It could, but that would reduce the information we get about the platform we're running on to that CI job. `lscpu` info is more widely valuable IMHO, especially given the variety of machines used by GH Actions.